docs/tutorials/absorption_in_BLR_with_cubepy.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    SUM_RATIO_LINES_BLR = 30.652
</pre><p>If this notebook has to be included in the docs I think it needs to be considerably improved, at the moment it has no commentary and it does not explain what its purpose is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cosimoNigro/agnpy/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Let's remove the notebook for now so we can go further. I would also like to develop other features, but it will be easier if the code for calculation absorption with cubepy is in the main repo. In the next step, I will add notebooks with explanations for the multi-shell BLR model.